designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/BasicInputImage.kt

```diff
@@ -81,35 +81,37 @@ internal fun <T> BasicInputImage(
         inputField = {
             when (uploadState) {
                 UploadState.ADD -> {
-                    Button(
-                        enabled = state != InputShellState.DISABLED,
-                        style = ButtonStyle.KEYBOARDKEY,
-                        text = addButtonText,
-                        icon = {
-                            Icon(
-                                imageVector = addButtonIcon,
-                                contentDescription = "Icon Button",
-                                tint = if (state != InputShellState.DISABLED) SurfaceColor.Primary else TextColor.OnDisabledSurface,
-                            )
+                    ButtonBlock(
+                        primaryButton = {
+                            Button(
+                                enabled = state != InputShellState.DISABLED,
+                                style = ButtonStyle.KEYBOARDKEY,
+                                text = addButtonText,
+                                icon = {
+                                    Icon(
+                                        imageVector = addButtonIcon,
+                                        contentDescription = "Icon Button",
```

Comment:
remember to pass the tint param otherwise it will show  the default purple color in the app